cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java

```diff
@@ -301,371 +292,85 @@ public void expirateRecords(long expirationTime) throws CygnusExpiratingError {
-                        } else {
-                            stringValue = "'" + value.toString() + "'";
-                        }
-                    } else {
-                        if (value.isJsonPrimitive()) {
-                            stringValue = "'" + value.getAsString() + "'";
-                        } else {
-                            stringValue = "'" + value.toString() + "'";
-                        }
-                    }
-                    if (first) {
-                        valuesForInsert += stringValue;
-                        first = false;
-                    } else {
-                        valuesForInsert += "," + stringValue;
-                    } // if else
-                } // while
-
-                valuesForInsert += ")";
-            } // for
-            
-            return valuesForInsert;
-        } // getValuesForInsert
-        
-        public String getFieldsForCreate() {
-            String fieldsForCreate = "(";
-            boolean first = true;
-            Iterator<String> it = aggregation.keySet().iterator();
-            
-            while (it.hasNext()) {
-                if (first) {
-                    fieldsForCreate += (String) it.next() + " text";
-                    first = false;
-                } else {
-                    fieldsForCreate += "," + (String) it.next() + " text";
-                } // if else
-            } // while
-            
-            return fieldsForCreate + ")";
-        } // getFieldsForCreate
-        
-        public String getFieldsForInsert() {
-            String fieldsForInsert = "(";
-            boolean first = true;
-            Iterator<String> it = aggregation.keySet().iterator();
-            while (it.hasNext()) {
-                if (first) {
-                    fieldsForInsert += (String) it.next();
-                    first = false;
-                } else {
-                    fieldsForInsert += "," + (String) it.next();
-                } // if else
-            } // while
-            return fieldsForInsert + ")";
-        } // getFieldsForInsert
-        
+        @Override
         public void initialize(NGSIEvent event) throws CygnusBadConfiguration {
-            service = event.getServiceForNaming(enableNameMappings);
-            servicePathForData = event.getServicePathForData();
+            super.initialize(event);
             servicePathForNaming = event.getServicePathForNaming(enableGrouping, enableNameMappings);
             entityForNaming = event.getEntityForNaming(enableGrouping, enableNameMappings, enableEncoding);
             entityType = event.getEntityTypeForNaming(enableGrouping, enableNameMappings);
             attribute = event.getAttributeForNaming(enableNameMappings);
-            dbName = buildDbName(service);
-            tableName = buildTableName(servicePathForNaming, entityForNaming, entityType, attribute);
+            setDbName(buildDbName(event.getServiceForNaming(enableNameMappings)));
```

Comment:
I'm a bit confused regarding getServiceForNaming() and getServicePathForNaming(). Should they have the same arguments? Any reason for omitting the serviceForNaming var?